### PR TITLE
Dress uniform and combo cover

### PR DESCRIPTION
**Pull request by Pericles1848**
#### Summary
Content "Dress uniform and combo cover"

#### Purpose of change
Army and military related items are currently limited mostly to items you'd find on soldiers in the field, like the army pants and jacket. While it makes sense that no one is wearing a service dress uniform in the field and thus no professions start with one and no soldier zombies in the field wear any, it seems strange that you can't find one on bases or in bunkers. 

#### Describe the solution
This PR adds a dress uniform and a combo cover, or peaked cap, that officers wear. The cover can be activated in a similar manner to the fedora, and produces a response about the character saluting.

#### Describe alternatives you've considered
Making a personal mod with the changes, associating items with an old guard quest, not implement combo cover and uniform.

#### Testing
I put the changed files in game, spawned the items with debug menu, and activated the cover.

#### Additional context
combocover
dressuniform

### CODE REVIEW

**Comment by faefux** [comment]
Nice commit, real cool. I'm unsure if the cap should have a salute use-action though as it's very specific and essentially just a roleplaying thing.

**Comment by Pericles1848** [comment]
> Nice commit, real cool. I'm unsure if the cap should have a salute use-action though as it's very specific and essentially just a roleplaying thing.

Thanks, my reasoning was if you can tip a fedora for roleplaying purposes you should be able to salute with a cap, I'm fine with taking it out though, it's a very minor thing

**Comment by faefux** [comment]
> > Nice commit, real cool. I'm unsure if the cap should have a salute use-action though as it's very specific and essentially just a roleplaying thing.
> 
> Thanks, my reasoning was if you can tip a fedora for roleplaying purposes you should be able to salute with a cap, I'm fine with taking it out though, it's a very minor thing

I didn't know you could do that with a fedora TBH. It isn't my call and is only my opinion, do not take it out on my behalf. :)

**Comment by Pericles1848** [comment]
Don't worry, I haven't taken it out, the last commit was just a fix to how it spawns.

File: data/mods/Aftershock/itemgroups/item_groups.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
       [ "afs_energy_saber_off", 3 ],
       [ "ftk93", 5 ],
       [ "afs_hardlight_longbow", 2 ],
+      [ "charge_rifle", 1 ],
```
Comment (Mom-Bun) [inline data/mods/Aftershock/itemgroups/item_groups.json:13]: Unrelated to this PR, requires removal.

Comment (Pericles1848) [reply data/mods/Aftershock/itemgroups/item_groups.json]: Should be fixed now.

**Comment by Maddremor** [comment]
I don't think the displayed names are to be capitalized.

File: data/json/items/armor/suits_protection.json [hunk 2]
```diff
@@ -846,6 +846,40 @@
...
+      { "pocket_type": "CONTAINER", "max_contains_volume": "250 ml", "max_contains_weight": "1 kg", "moves": 120 },
+      { "pocket_type": "CONTAINER", "max_contains_volume": "250 ml", "max_contains_weight": "1 kg", "moves": 120 },
+      { "pocket_type": "CONTAINER", "max_contains_volume": "350 ml", "max_contains_weight": "1 kg", "moves": 80 },
+      { "pocket_type": "CONTAINER", "max_contains_volume": "350 ml", "max_contains_weight": "1 kg", "moves": 80 },
+      { "pocket_type": "CONTAINER", "max_contains_volume": "250 ml", "max_contains_weight": "1 kg", "moves": 100 },
+      { "pocket_type": "CONTAINER", "max_contains_volume": "250 ml", "max_contains_weight": "1 kg", "moves": 100 }
+    ],
+    "warmth": 25,
+    "material_thickness": 0.3,
+    "flags": [ "VARSIZE", "SUPER_FANCY", "POCKETS" ]
```
Comment (Maleclypse) [inline data/json/items/armor/suits_protection.json:881]: I think let's move this down to FANCY from SUPER_FANCY.  I'm basing this off Kevin's recent comments around auditing FANCY items.

Comment (actual-nh) [inline data/json/items/armor/suits_protection.json:881]:
```suggestion
    "flags": [ "VARSIZE", "FANCY", "POCKETS" ]
```

Comment (Pericles1848) [reply data/json/items/armor/suits_protection.json]: That seems fair

File: data/json/items/tool_armor.json [hunk 3]
```diff
@@ -114,6 +114,33 @@
     "environmental_protection": 1,
     "flags": [ "BLIND" ]
   },
+  {
+    "type": "TOOL_ARMOR",
+    "category": "clothing",
+    "id": "combo_cover",
+    "name": { "str": "Combo cover" },
```
Comment (casswedson) [inline data/json/items/tool_armor.json:121]:
```suggestion
    "name": { "str": "combo cover" },
```

**Comment by Maleclypse** [comment]
Please open a new PR for this targeting the experimental branch.  Was closed as a side effect of deleting the 0.F -dev branch

**Comment by faefux** [comment]
Hi, @Pericles1848 . Do you have plans to re-submit this?
If you don't feel like it would you be fine with me doing it? I felt this was a nice addition.

**Comment by kevingranade** [comment]
Obsoleted by #51835

**Pull request closed without merging**